lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARFDebugMap.cpp

```diff
@@ -716,6 +718,27 @@ bool SymbolFileDWARFDebugMap::ParseDebugMacros(CompileUnit &comp_unit) {
   return false;
 }
 
+void SymbolFileDWARFDebugMap::ForEachSymbolFile(
+    std::string description,
+    std::function<IterationAction(SymbolFileDWARF &)> closure) {
+  const size_t num_oso_idxs = m_compile_unit_infos.size();
+  Progress progress(std::move(description), "", num_oso_idxs,
+                    /*debugger=*/nullptr,
+                    /*minimum_report_time=*/std::chrono::milliseconds(20));
```

Comment:
It might be nice to have a setting in lldb for the time between notifcations. 500 per second is a bit high IMHO, but it would be nice to have a setting and also use it in lldb/source/Plugins/SymbolFile/DWARF/ManualDWARFIndex.cpp in a follow up patch?

Reply:
I used 20ms because that's what Pavel used in the [manual DWARF index](https://github.com/llvm/llvm-project/blob/4485e25dd2a57be1ee504b4dd863a1e140f5084c/lldb/source/Plugins/SymbolFile/DWARF/ManualDWARFIndex.cpp#L84). That's 50 times per second, which is still fast but not as bad as 500. That said, having a setting that both these things query doesn't sound like a bad idea. It could even be the default for all progress events. WDYT?